packages/flutter_test/lib/src/test_text_input.dart

```diff
@@ -6,25 +6,35 @@ import 'dart:async';
 import 'dart:convert';
 import 'dart:typed_data';
 
-import 'package:flutter_test/flutter_test.dart';
 import 'package:flutter/services.dart';
 
+import 'widget_tester.dart';
+
 const String _kTextInputClientChannel = 'flutter/textinputclient';
 
-class MockTextInput {
+/// A testing stub for the system's popup keyboard.
```

Comment:
Ha! I wasn't sure what to call it so I asked the web about "Android SDK Keyboard" and the first thing I saw was https://developer.android.com/reference/android/inputmethodservice/Keyboard.Key.html which refers to it as the popup keyboard. That said, I'm happy to call it the onscreen keyboard. Obviously I don't know what its official name is :-).